packages/rum-core/src/common/utils.js

```diff
@@ -135,14 +135,14 @@ function isPlatformSupported() {
 }
 
 /**
- * Convert values of the tag to be string to be compatible
+ * Convert values of the Tag/Label to be string to be compatible
  * with the apm server prior to <6.7 version
  *
  * TODO: Remove string conversion in the next major release since
  * support for boolean and number in the APM server has landed in 6.7
  * https://github.com/elastic/apm-server/pull/1712/
  */
-function setTag(key, value, obj) {
+function setLabel(key, value, obj) {
```

Comment:
the java agent has added support for boolean/number values in the labels payload along with deprecating `addTags`. Should we also do that as part of this PR? @jahtalab /cc @felixbarny

Reply:
I think adding support for boolean/number is a breaking change since it those values were not supported by APM server prior to 6.7 release. IMO we should do them in a major version.

Reply:
I've created [this issue](https://github.com/elastic/apm-agent-rum-js/issues/272) to address this!

Reply:
Cool, let's do it as part of 5.0.x then